expected/disable_hashagg_1.out

```diff
@@ -29,6 +29,8 @@ SELECT hll_set_output_version(1);
 (1 row)
 
 CREATE TABLE tt1(id int, col_1 int, sd hll);
+NOTICE:  using default RANDOM distribution since no distribution was specified
+HINT:  Consider including the 'DISTRIBUTED BY' clause to determine the distribution of rows.
```

Comment:
this can delete too?I think

Reply:
There are some other diffs in this output file. I think removing these two lines by hand is not appropriate.